src/routes/safe/components/Settings/Advanced/dataFetcher.ts

```diff
@@ -0,0 +1,33 @@
+import { List } from 'immutable'
+import { TableColumn } from 'src/routes/safe/components/tableTypes'
+import { ModulePair } from 'src/routes/safe/store/models/safe'
+
+export const MODULES_TABLE_ADDRESS_ID = 'address'
+export const MODULES_TABLE_ACTIONS_ID = 'actions'
+
+export const getModuleData = (modulesList: List<ModulePair>): List<{ [MODULES_TABLE_ADDRESS_ID]: ModulePair }> => {
+  return modulesList?.map((modules) => ({
```

Comment:
why optional chaining? so if `moduleList` can be undefined, the type should be changed respecting to that. Also, if moduleList is undefined, it would return undefined and that means return type should be changed too